// C-ERR

func main() {
    (5 + 5) = "value"
}